announcements/2020-Week-42/announcement.md

```diff
@@ -0,0 +1,15 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [NGINX Ingress Controller app v1.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1100---2020-10-07) upgrades ingress-nginx-controller from `v0.35.0` to `v0.40.2` improving performance and reliability.
+2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. SEs are working with customers to prepare.)
+3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus`
```

Comment:
Was there a change in the prometheus operator version?

Reply:
in the linked [changelog](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15), it says, "prometheus-operator v0.38.1 -> v0.42.1"

that's a good point... Is that the main thing customers would be looking for? If yes, then that should be included here... will talk w team